### PR TITLE
Alt text

### PR DESCRIPTION
**Pull request by AverageHelper**
Added a command to get the alt text from a message.

I'm not sold on the formatting, but this might be a good-enough first go for us to iterate on.

### CODE REVIEW

**Review by SpencerHastings — APPROVED** [review]
🦀

File: src/commands/contextMenu/altText.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+				`None of the ${attachments.length} attachments in this message contains a text description.`
+			);
+		}
+
+		let response = '## Image Descriptions';
+		let idx = 1;
+		for (const attachment of attachments) {
+			const description = attachment.description?.trim();
+			if (description) {
+				response += `\n### Image ${idx}\n> ${description}`;
```
Comment (Plyb) [inline src/commands/contextMenu/altText.ts:35]: are images the only kind of attachment that has descriptions?

Comment (AverageHelper) [reply src/commands/contextMenu/altText.ts:35]: As far as I know. I've tried .exe, .pdf, and .mp4. Gif and other image formats get a Description field.

Comment (AverageHelper) [reply src/commands/contextMenu/altText.ts:35]: We could change the verbiage to "Item X", in case Discord expands the scope later.

Comment (Plyb) [reply src/commands/contextMenu/altText.ts:35]: I think something like that, or like "attachment x" would be good, but if it's working for now, I don't think it's enough to block the ticket going through

**Review by Plyb — APPROVED** [review]

**Pull request merged**